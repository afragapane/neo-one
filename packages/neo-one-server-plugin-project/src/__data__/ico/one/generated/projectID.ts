// tslint:disable
/* eslint-disable */
/**
 * @projectID d56e9014-bafc-40c0-868f-19701f1a2dd0
 */

export const projectID = 'd56e9014-bafc-40c0-868f-19701f1a2dd0';
