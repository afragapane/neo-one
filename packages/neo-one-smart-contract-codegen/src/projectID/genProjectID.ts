export const genProjectID = ({ projectID }: { readonly projectID: string }) => ({
  ts: `
/**
 * @projectID ${projectID}
 */

export const projectID = '${projectID}';
`,
  js: `
/**
 * @projectID ${projectID}
 */

export const projectID = '${projectID}';
`,
});
