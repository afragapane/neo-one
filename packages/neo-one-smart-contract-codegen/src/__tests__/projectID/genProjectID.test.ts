import { genProjectID } from '../../projectID';

describe('genProjectID', () => {
  test('Token', () => {
    expect(
      genProjectID({
        projectID: 'foobar',
      }),
    ).toMatchSnapshot();
  });
});
