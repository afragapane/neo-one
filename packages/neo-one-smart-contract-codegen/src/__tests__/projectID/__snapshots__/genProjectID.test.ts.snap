// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`genProjectID Token 1`] = `
Object {
  "js": "
/**
 * @projectID foobar
 */

export const projectID = 'foobar';
",
  "ts": "
/**
 * @projectID foobar
 */

export const projectID = 'foobar';
",
}
`;
